components/src/dynamo/planner/tests/unit/test_planner_sweep_config_provider.py

```
@@ -537,6 +540,257 @@ def test_provider_owns_its_adapter_and_hook_abi_versions() -> None:
+        + "\n"
+        + json.dumps(
+            {
+                "event": {
+                    "event_type": "request_end",
+                    "agent_context": {"session_id": "s"},
+                }
+            }
+        )
+        + "\n"
+    )
+    with pytest.raises(ValueError, match="planner.policy=disabled"):
```

Ruff (0.16.2)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)